src/github/client/installation-token-cache.ts

```diff
@@ -18,13 +18,13 @@ export default class InstallationTokenCache {
 	 * @param maxTokens the max number of tokens that should be in the cache at any time. If the size of the cache is about to go above this
 	 * number, the least recently used tokens are evicted from the cache.
 	 */
-	constructor(maxTokens: number) {
-		this.installationTokenCache = new LRUCache<number, AuthToken>({ max: maxTokens });
+	constructor() {
+		this.installationTokenCache = new LRUCache<number, AuthToken>({ max: 1000 });
```

Comment:
Why did we set it to constant here?

Reply:
Because it never changed anywhere.  No point of having it as a variable.

Reply:
/nit we can extract it into constant to improve clarity